Orm/Xtensive.Orm/Orm/Linq/Materialization/ExpressionMaterializer.cs

```diff
@@ -63,9 +67,10 @@ protected override Expression VisitFullTextExpression(FullTextExpression express
     {
       var rankMaterializer = Visit(expression.RankExpression);
       var entityMaterializer = Visit(expression.EntityExpression);
-      var constructorInfo = typeof (FullTextMatch<>)
+      var constructorInfo = WellKnownOrmTypes.FullTextMatchOfT
         .MakeGenericType(expression.EntityExpression.Type)
-        .GetConstructor(BindingFlags.NonPublic | BindingFlags.Instance, new[] {typeof (double), expression.EntityExpression.Type});
+        .GetConstructor(BindingFlags.NonPublic | BindingFlags.Instance,
+          new object[] {WellKnownTypes.Double, expression.EntityExpression.Type});

```

Comment:
We can make this array pre-allocated in static var.

Reply:
But it depends on `expression.EntityExpression.Type`

Reply:
Yes, my mistake.